templates/hackathon-store/example_prompt.md

```
@@ -0,0 +1,60 @@
+## Examples
+/poap 0x1234567890123456789012345678901234567890
+/list
+/faucet 0x1234567890123456789012345678901234567890 sepolia
+/networks
+/update
+
+### Goodies
+- When greeted for the first time, give the full menu.
+- The user can select the option by number or name
+- Once the option is selected confirm the order
+## Response Scenarios:
```

Markdownlint (0.35.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)